app/views/matches/_confirmed.html.erb

```diff
@@ -16,3 +16,7 @@
     </p>
   </div>
 <% end %>
+
+<p class="mt-4">
+  En cas d'empêchement, vous pouvez joindre le centre de vaccination au numéro suivant : <%= @match.vaccination_center.phone_number %>
```

Comment:
We could add a link `tel:` so the user can directly click on the phone number and call the vaccination center

Reply:
Thank you. It's done !